databricks/koalas/series.py

```diff
@@ -154,6 +154,54 @@ def spark_type(self):
         """ Returns the data type as defined by Spark, as a Spark DataType object."""
         return self.schema.fields[-1].dataType
 
+    # TODO: arg should support Series and dictionary mapping too.
+    def map(self, arg):
```

Comment:
can you add the dictionary mapping? I was just needing it yesterday and this is a big ask from @anabranch as well.

Reply:
Let me take a look